### DISCUSSION TITLE
分享按钮无法复制链接

### DISCUSSION THREAD

**Discussion by youyou-sudo** [libsgh/PanIndex#282] [General]
页面样式问题，会被遮挡
![image](https://user-images.githubusercontent.com/60727068/162653872-c3aa012a-a585-4572-b09f-7219aa4feab8.png)

**Comment by libsgh** [comment]
忘记是不是已经修复这个问题了...

**Reply by youyou-sudo** [reply]
> 忘记是不是已经修复这个问题了...

已在3.0.7版本修复
